releasenotes/notes/remote-config-http-client-backoff-d9acbc7bc24b4ca7.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    The Remote Configuration client will do exponential backoff
+    for subsequent requests when errors are encountered to preserve
+    service stability.
```

Comment:
Is remote configuration public ? I could not find anything in our release note and the configuration isn't in the config template (which make me this it's still internal/beta)

Release notes are used to generate the CHANGELOG which is meant for users not dev. We add item to it only if we want any user/customer to use the new feature. I guess this PR should not have a release note. You can use the `changelog/no-changelog` Github label to pass the CI.

Reply:
It's still only internal.